### ISSUE TITLE
Seeing 'Update Pending' issues in the log files and replicator is replicating contents

### ISSUE THREAD

**Issue by sudarshanb** [projectkudu/AzureSiteReplicator#9]
Hi,

We installed replicator like month back and it was working fine. But all of sudden its started showing below messages in log:

2015/05/29-20:32:09:37  Verbose Source filePath (D:\home\site\wwwroot\media\1434\gluten-free-ginger-berry-crisp.txt) does not match destination (site name\Media\1434\gluten-free-ginger-berry-crisp.txt) differing in attributes (lastWriteTime['05/28/2015 18:45:50','01/01/1980 06:00:00']). Update pending.
2015/05/29-20:32:09:37  Verbose Source filePath (D:\home\site\wwwroot\media\1435\jamie_barry_boy_girl.jpg) does not match destination (site name\Media\1435\jamie_barry_boy_girl.jpg) differing in attributes (lastWriteTime['05/28/2015 18:45:50','01/01/1980 06:00:00']). Update pending.
2015/05/29-20:32:09:37  Verbose Source filePath (D:\home\site\wwwroot\media\1435\jamie_barry_boy_girl_thumb.jpg) does not match destination (site name\Media\1435\jamie_barry_boy_girl_thumb.jpg) differing in attributes (lastWriteTime['05/28/2015 18:45:50','01/01/1980 06:00:00']). Update pending.

And its no longer syncing the contents to other site.

Any ideas about what is happening?

Thanks for your help!

**Comment by davidebbo** [comment]
Note sure. I'd try using Kudu console in both sites to look at one of those files in both source and target (including running `attrib` on it). Are these files that have changed and that you would have expected to get updated?

**Comment by sudarshanb** [comment]
Yes, we updated those in the primary site and was expecting to flow those changes to secondary site.
Any idea how to fix these errors?

**Comment by davidebbo** [comment]
Did you try my previous suggestion?
